src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -695,10 +695,15 @@ public async Task SslStream_NestedAuth_Throws()
         [InlineData(false, true)]
         [InlineData(false, false)]
         [InlineData(true, true)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/68206", TestPlatforms.Android)]
         public async Task SslStream_TargetHostName_Succeeds(bool useEmptyName, bool useCallback)
         {
             string targetName = useEmptyName ? string.Empty : Guid.NewGuid().ToString("N");
+            if (PlatformDetection.IsAndroid && !useEmptyName)
```

Comment:
I would probably use something like `$GUID.test.org` or `$GUID.SslStream_TargetHostName_Succeeds.net` or   use `localhost` on all platforms. Using GUID makes it sometimes easier to debug e.g. you can see unique SNI in packet capture and it avoids side effects with TLS resume and credential cache. Other than that the actual string does not matter AFAIK.